content/documentation/3-stream-developer-guides/2-streams/1-standalone-stream-rabbitmq.md

```diff
@@ -0,0 +1,229 @@
+---
+path: 'stream-developer-guides/streams/standalone-stream/'
+title: 'Standalone Streaming Application Development on RabbitMQ'
+description: 'Create a simple stream processing application on RabbitMQ'
+---
+
+# Stream Processing with Apache Kafka
+
+In this guide we will develop a stream processing application and deploy it to Cloud Foundry, Kubernetes, and on your local machine.
+In another guide, we will deploy the stream processing application using Data Flow.
+
+We will start from initializr and create two Spring Cloud Stream applications.
+Note for CF we need a manifest, for k8s we need a service/deployment yaml.
+
+## Development
+
+### Initialzr
+
+Go to [Spring Cloud Stream Initializr](https://start-scs.cfapps.io/) and
+
+- Generate a `Maven Project` With Spring Boot `2.1.4` (Choose the version of your choice)
```

Comment:
Try to avoid specifying a specific release i.e. 2.1.4. For example `Spring Boot current release`